crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -320,13 +321,26 @@ impl RefreshTask {
         let dataset_name = self.dataset_name.clone();
         let filter_converter = self.get_filter_converter(refresh);
 
+        let component_type = if self.is_view_acceleration() {
+            "view"
+        } else {
+            "dataset"
+        };
+
         if is_spice_internal_dataset(&dataset_name) {
-            tracing::debug!("Loading data for dataset {dataset_name}");
+            tracing::debug!("Loading data for {component_type} {dataset_name}");
+        } else {
+            tracing::info!("Loading data for {component_type} {dataset_name}");
+        }
+
+        if self.is_view_acceleration() {
```

Comment:
Similarly we can have a method `update_status()` that just takes the component status.

Reply:
Updated, thank you - . Ensured that all updates are done via single  path`set_refresh_status` -> `update_component_status` that updates status correctly. Minimized changes to simplify review. We can have next iteration to refactor/rename naming w/o changing functionality next.